[Common] 페이지별 heading 정리 및 파일 최적화

ingbinsee opened the pull request:
## 🔍️ 이 PR을 통해 해결하려는 문제가 무엇인가요?

> 어떤 기능을 구현한건지, 이슈 대응이라면 어떤 이슈인지 PR이 열리게 된 계기와 목적을 Reviewer 들이 쉽게 이해할 수 있도록 적어 주세요
> 일감 백로그 링크나 다이어그램, 피그마를 첨부해도 좋아요

- 웹 접근성 개선
- 페이지의 흐름 이해 용이
- 성능 개선

## ✨ 이 PR에서 핵심적으로 변경된 사항은 무엇일까요?

> 문제를 해결하면서 주요하게 변경된 사항들을 적어 주세요

- 페이지별 heading 정리
- 이미지, PDF 파일 최적화

## 🔖 핵심 변경 사항 외에 추가적으로 변경된 부분이 있나요?

> 없으면 "없음" 이라고 기재해 주세요

- 없음

## 🙏 Reviewer 분들이 이런 부분을 신경써서 봐 주시면 좋겠어요

> 개발 과정에서 다른 분들의 의견은 어떠한지 궁금했거나 크로스 체크가 필요하다고 느껴진 코드가 있다면 남겨주세요

- 없음

## 🩺 이 PR에서 테스트 혹은 검증이 필요한 부분이 있을까요?

> 테스트가 필요한 항목이나 테스트 코드가 추가되었다면 함께 적어주세요

- 없음

### 📌 PR 진행 시 이러한 점들을 참고해 주세요

- Reviewer 분들은 코드 리뷰 시 좋은 코드의 방향을 제시하되, 코드 수정을 강제하지 말아 주세요.
- Reviewer 분들은 좋은 코드를 발견한 경우, 칭찬과 격려를 아끼지 말아 주세요.
- Review는 특수한 케이스가 아니면 Reviewer로 지정된 시점 기준으로 3일 이내에 진행해 주세요.
- Comment 작성 시 Prefix로 P1, P2, P3 를 적어 주시면 Assignee가 보다 명확하게 Comment에 대해 대응할 수 있어요
  - P1 : 꼭 반영해 주세요 (Request Changes) - 이슈가 발생하거나 취약점이 발견되는 케이스 등
  - P2 : 반영을 적극적으로 고려해 주시면 좋을 것 같아요 (Comment)
  - P3 : 이런 방법도 있을 것 같아요~ 등의 사소한 의견입니다 (Chore)

#

---

## 📝 Assignee를 위한 CheckList

- [ ] To-Do Item

ingbinsee committed:
🐛 [Fix] : max-width 설정

sectionTitle의 크기를 조정하기 위해 max-width(1320px)를 설정하였습니다.

ingbinsee committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화

ingbinsee committed:
♻️ [Refactor] : 페이지별 heading 정리

heading 요소를 흐름에 맞게 변경하였습니다.

Related to: #144

ingbinsee committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화

ingbinsee committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화

ingbinsee committed:
♻️ [Refactor] : 페이지별 heading 정리

heading 요소를 흐름에 맞게 변경하였습니다.

Related to: #144

ingbinsee committed:
♻️ [Refactor] : SectionTitle 정렬 변경 및 코드 오류 수정

병합 과정에서 SectionTitle이 중앙 정렬로 변경된 것을 왼쪽 정렬로 변경하였습니다.
에러가 나는 코드를 줄바꿈하여 오류를 해결하였습니다.

dani-day reviewed (approved):
이견 없습니다 :)

ingbinsee committed:
♻️ [Refactor] : 이미지, PDF 파일 최적화

용량이 큰 이미지, PDF 파일의 용량을 줄여 성능을 개선하고자 하였습니다.

Related to: #148

ingbinsee commented:
파일 최적화를 위해 변경한 파일 및 코드 업데이트하였습니다. 확인 부탁드립니다.

kelvin-chihyun commented on src/components/Contact/Agreement.jsx:
```diff
@@ -9,7 +9,7 @@ function Agreement({ state, setState }) {
   return (
     <fieldset className={styles.agreementText}>
-      <h4>개인정보 활용동의</h4>
+      <h3>개인정보 활용동의</h3>
```
P3: 아주 사소하지만,, bold처리로 두는 것이 어떨까 싶습니다.

ingbinsee replied:
&lt;h4&gt;에서 &lt;h3&gt;으로 변경하면서 스타일이 깨졌네요. 확인 감사합니다.
반영하였습니다.

kelvin-chihyun commented on src/components/Home/Solutions.jsx:
```diff
@@ -119,8 +120,11 @@ function Solutions() {
             <>
-              <h4 className={`${styles.title}`}>{solutionData[0].title}</h4>
-              <p className={styles.summary}>{solutionData[0].desc}.</p>
+              <p className={`${styles.title}`}>{solutionData[0].title}</p>
```
P1: 제가 작성했던 컴포넌트지만, 만약 solution 종류를 h4로 두지 않는다면, 아래 사진에서 Products.jsx 내부에 있는 Solutions, smart machine, smart device도 헤딩에서 제외를 해야할 것 같습니다!!
(h4를 포함시킨다면, industries.jsx 내부 4개 industry도 헤딩이 되어야할 것 같네요!)

![Screenshot 2023-12-20 at 1 45 11 PM](https://github.com/ReOpenSG/ReOpenSG/assets/101504272/8f37d0fe-47b7-4053-8e26-b229ed37a20b)

ingbinsee replied:
미처 이 부분을 체크하지 못했네요. 수정했습니다.

kelvin-chihyun reviewed (changes requested):
고생하셨습니다! P1 의견은 조치 부탁드립니다!

ingbinsee committed:
🌈 [Design] : <h4>에서 <h3>으로 변경하면서 깨진 스타일 수정

heading을 변경하면서 깨진 스타일을 수정하였습니다.

Resolves: #149

ingbinsee committed:
♻️ [Refactor] : products 리스트 heading 제외

solutions section을 참고하여 products section의 products 리스트를 heading에서 제외하였습니다.

Resolves: #149

kelvin-chihyun reviewed (approved):
확인했습니다. 고생하셨습니다.
